ansible_runner/runner_config.py

```diff
@@ -186,7 +186,9 @@ def prepare(self):
         if self.module and self.playbook:
             raise ConfigurationError("Only one of playbook and module options are allowed")
         if not os.path.exists(self.artifact_dir):
-            os.makedirs(self.artifact_dir, mode=0o700)
+            original_umask = os.umask(0)
```

Comment:
I was surprised that the permissions passed to `os.makedirs` seemed to be overridden by the system's default umask without this. Would it be better to do an explicit chmod after creating the directory instead?